Update resources-data.ts

randy-r opened the pull request:
Hello Stencil Team,

Please let me know if I can add those resources to the site.
Awesome work by the way!

Thanks,
Randy.

tanner-reits commented on src/components/resources-page/resources-data.ts:
```diff
@@ -160,6 +162,20 @@ export const RESOURCES = {
+    },
+    {
+      title: 'Stedux',
+      description: 'Example website for the using the Stedux library',
```
```suggestion
      description: 'Example website for using the Stedux library',
```

tanner-reits commented on src/components/resources-page/resources-data.ts:
```diff
@@ -160,6 +162,20 @@ export const RESOURCES = {
+    },
+    {
+      title: 'Stenimator',
+      description: 'Example website for the using the Stenimator library',
```
```suggestion
      description: 'Example website for using the Stenimator library',
```

rwaskiewicz commented on src/components/resources-page/resources-data.ts:
```diff
@@ -29,7 +29,9 @@ export const RESOURCES = {
-    { title: 'Soft Components (Neumorphism design inspired web components)', url: 'https://soft.components.rocks' }
+    { title: 'Soft Components (Neumorphism design inspired web components)', url: 'https://soft.components.rocks' },
+    { title: 'Stedux: Redux for Stencil', url: 'https://github.com/randy-r/stedux' },
+    { title: 'Stenimator: Animation coordintor for elements leaving the DOM', url: 'https://github.com/randy-r/stenimator' }
```
```suggestion
    { title: 'Stenimator: Animation coordinator for elements leaving the DOM', url: 'https://github.com/randy-r/stenimator' }
```

rwaskiewicz reviewed (changes requested):
Thanks @randy-r! 

The team and I had a few small nitpicks we found while reviewing the PR. Could you please take a look when you get a chance? Thanks again!

randy-r commented:
Hi @rwaskiewicz !

I should thank you and the team!
I have addressed your suggestions and adjusted the image size as they were a bit off.

Cheers!

rwaskiewicz reviewed (approved):
LGTM! Thanks!
